SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -711,6 +716,51 @@ bool FSpatialGDKEditorToolbarModule::GenerateDefaultLaunchConfig(const FString&
 	return false;
 }
 
+bool FSpatialGDKEditorToolbarModule::GenerateDefaultWorkerJson()
+{
+	if (const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>())
+	{
+		const FString WorkerJsonDir = FSpatialGDKServicesModule::GetSpatialOSDirectory(TEXT("workers/unreal"));
+		const FString TemplateWorkerJsonPath = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(TEXT("SpatialGDK/Extras/templates/WorkerJsonTemplate.json"));
+
+		const FSpatialLaunchConfigDescription& LaunchConfigDescription = SpatialGDKEditorSettings->LaunchConfigDesc;
+		for (const FWorkerTypeLaunchSection& Worker : LaunchConfigDescription.ServerWorkers)
+		{
+			FString JsonPath = FPaths::Combine(WorkerJsonDir, FString::Printf(TEXT("spatialos.%s.worker.json"), *Worker.WorkerTypeName.ToString()));
+			if (!FPaths::FileExists(JsonPath))
+			{
+				UE_LOG(LogSpatialGDKEditorToolbar, Verbose, TEXT("Could not worker json at %s"), *JsonPath);
```

Comment:
Could not worker?

Reply:
Why waste time say lot word when few word do trick?